dnf5.spec

```diff
@@ -2,14 +2,35 @@
 %global project_version_minor 0
 %global project_version_patch 0
 
-Name:           libdnf5
+Name:           dnf5
 Version:        %{project_version_major}.%{project_version_minor}.%{project_version_patch}
 Release:        0~pre%{?dist}
 Summary:        Package management library
 License:        LGPLv2.1+
```

Comment:
Also Summary and License should be from dnf5 subpackage:
```
Summary:        Command-line package manager
License:        GPLv2+
```

And to library subpackages (libdnf5, libdnf5-cli, -devel, and language bindings) should go `License: LGPLv2.1+`